ironfish/src/memPool/memPool.test.ts

```diff
@@ -10,12 +11,13 @@ import { TransactionVersion } from '../primitives/transaction'
 import {
   createNodeTest,
   useAccountFixture,
+  useBlockWithCustomTxs,
   useBlockWithTx,
   useMinerBlockFixture,
   useTxFixture,
 } from '../testUtilities'
 import { Account } from '../wallet'
-import { getFeeRate } from './feeEstimator'
+import { getFeeRate, getPreciseFeeRate } from './feeEstimator'
```

Comment:
Is `getFeeRate` effectively deprecated now, or do we still need it somewhere?

Reply:
We still use it in the CLI. Planning on fully removing it in future PR